### PR TITLE
release: 1.4.1

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

**Comment by Odyssey346** [comment]
lgtm

File: src/commands/changelog.ts [hunk 1]
```diff
@@ -4,10 +4,15 @@ exports.run = async (client, message) => {
...
-        title: "**V1.4.0**",
+        title: "**V1.4.1**",
         description:
           "A **lot** has changed in 1.4, and the team has almost entirely changed, so this isn't an exhaustive list.",
         fields: [
+          {
+            name: "1.4.1 changes",
+            value: 
+              "- Added the `fact` command! Use this command to find out some fun facts about Cytrus-RE.\n- Added embeds to `c.math` and `c.colourutil` (when you haven't specified an action).\n- Fixed a few bugs (specifically, the version number was outdated in a few places + some code fixes)."
+          },
```
Comment (Odyssey346) [inline src/commands/changelog.ts:15]: this should be at the bottom

File: src/commands/changelog.ts [hunk 2]
```diff
@@ -40,7 +45,7 @@ exports.run = async (client, message) => {
         ],
         footer: {
           text:
-            "Cytrus-RE 1.4.0 was made possible by Rexowogamer#1183, Odyssey346#9848, Devnol#9366 and outside contributors. Thank you!",
+            "Cytrus-RE 1.4.0/1.4.1 was made possible by Rexowogamer#1183, Odyssey346#9848, Devnol#9366 and outside contributors. Thank you!",
```
Comment (Odyssey346) [inline src/commands/changelog.ts:48]: this sounds weird for some reason. maybe make it say "Cytrus-RE 1.4.0 & 1.4.1" instead

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/commands/changelog.ts (hunk 1 above)
Comment (Rexogamer) [reply src/commands/changelog.ts:15]: no, since they're the latest changes and because it's a hotfix they should really be at the top imo

Comment (Odyssey346) [reply src/commands/changelog.ts:15]: nah i think it should be at the bottom in a "hotfixes" section or something like that.

**Comment by Odyssey346** [comment]
LGTM, close the & / conversation and try to see if we can mutually agree on something for the positioning on the changelog

File: src/commands/changelog.ts (hunk 1 above)
Comment (Rexogamer) [reply src/commands/changelog.ts:15]: I'd expect it to be at the top if I were a user because, well, it's the newest

Comment (Odyssey346) [reply src/commands/changelog.ts:15]: well atleast put in a section for hotfixes because we're probably gonna work a lot on 1.4

**Pull request merged**